Training with a single process on 1 device (cuda:0).
WARNING: No pretrained configuration specified for MONet_T_no_multistage_no_conv model. Using a default. Please add a config to the model pretrained_cfg registry or pass explicitly.
Model MONet_T_no_multistage_no_conv created, param count:24779022
Data processing configuration for current model + dataset:
	input_size: (3, 32, 32)
	interpolation: bicubic
	mean: (0.485, 0.456, 0.406)
	std: (0.229, 0.224, 0.225)
	crop_pct: 0.875
	crop_mode: center
AMP not enabled. Training in float32.
/home/sharipov/monet/venv/lib64/python3.9/site-packages/torch/utils/data/dataloader.py:558: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
Scheduled epochs: 300. LR stepped per epoch.
Train: 0 [   0/625 (  0%)]  Loss: 2.30 (2.30)  Time: 1.718s,   37.25/s  (1.718s,   37.25/s)  LR: 1.000e-05  Data: 1.079 (1.079)
Train: 0 [  50/625 (  8%)]  Loss: 2.20 (2.27)  Time: 0.142s,  452.29/s  (0.174s,  367.72/s)  LR: 1.000e-05  Data: 0.014 (0.035)
Train: 0 [ 100/625 ( 16%)]  Loss: 2.09 (2.23)  Time: 0.141s,  453.67/s  (0.158s,  404.57/s)  LR: 1.000e-05  Data: 0.014 (0.025)
Train: 0 [ 150/625 ( 24%)]  Loss: 2.26 (2.21)  Time: 0.142s,  450.05/s  (0.153s,  418.79/s)  LR: 1.000e-05  Data: 0.014 (0.022)
Train: 0 [ 200/625 ( 32%)]  Loss: 2.12 (2.20)  Time: 0.143s,  448.07/s  (0.150s,  426.33/s)  LR: 1.000e-05  Data: 0.015 (0.020)
Train: 0 [ 250/625 ( 40%)]  Loss: 2.16 (2.19)  Time: 0.141s,  452.89/s  (0.148s,  431.01/s)  LR: 1.000e-05  Data: 0.014 (0.019)
Train: 0 [ 300/625 ( 48%)]  Loss: 2.11 (2.18)  Time: 0.142s,  449.69/s  (0.147s,  434.11/s)  LR: 1.000e-05  Data: 0.014 (0.018)
Train: 0 [ 350/625 ( 56%)]  Loss: 1.99 (2.17)  Time: 0.142s,  450.90/s  (0.147s,  436.44/s)  LR: 1.000e-05  Data: 0.013 (0.018)
Train: 0 [ 400/625 ( 64%)]  Loss: 2.05 (2.17)  Time: 0.140s,  455.57/s  (0.146s,  437.53/s)  LR: 1.000e-05  Data: 0.014 (0.017)
Train: 0 [ 450/625 ( 72%)]  Loss: 2.12 (2.16)  Time: 0.142s,  450.92/s  (0.146s,  439.00/s)  LR: 1.000e-05  Data: 0.021 (0.017)
Train: 0 [ 500/625 ( 80%)]  Loss: 2.18 (2.16)  Time: 0.141s,  452.40/s  (0.145s,  440.02/s)  LR: 1.000e-05  Data: 0.014 (0.017)
Train: 0 [ 550/625 ( 88%)]  Loss: 2.11 (2.15)  Time: 0.143s,  446.40/s  (0.145s,  441.01/s)  LR: 1.000e-05  Data: 0.014 (0.017)
Train: 0 [ 600/625 ( 96%)]  Loss: 2.21 (2.15)  Time: 0.143s,  446.25/s  (0.145s,  441.85/s)  LR: 1.000e-05  Data: 0.014 (0.016)
Test: [   0/156]  Time: 1.581 (1.581)  Loss:   1.761 ( 1.761)  Acc@1:  42.188 ( 42.188)  Acc@5:  85.938 ( 85.938)
Test: [  50/156]  Time: 0.040 (0.070)  Loss:   1.940 ( 1.880)  Acc@1:  26.562 ( 32.598)  Acc@5:  82.812 ( 84.896)
Test: [ 100/156]  Time: 0.040 (0.055)  Loss:   1.748 ( 1.878)  Acc@1:  34.375 ( 32.720)  Acc@5:  89.062 ( 84.870)
Test: [ 150/156]  Time: 0.032 (0.050)  Loss:   1.907 ( 1.876)  Acc@1:  29.688 ( 33.092)  Acc@5:  87.500 ( 85.192)
Test: [ 156/156]  Time: 0.075 (0.050)  Loss:   1.954 ( 1.876)  Acc@1:  18.750 ( 33.110)  Acc@5:  81.250 ( 85.170)
Traceback (most recent call last):
  File "/home/sharipov/monet/train.py", line 1190, in <module>
    main()
  File "/home/sharipov/monet/train.py", line 900, in main
    weight_norms(model)
  File "/home/sharipov/monet/train.py", line 845, in weight_norms
    current_s_norm = spectral_norm(param.data)
  File "/home/sharipov/monet/train.py", line 837, in spectral_norm
    u, s, v = torch.svd(weight, compute_uv=False)
RuntimeError: linalg.svd: input should have at least 2 dimensions, but has 1 dimensions instead
[2024-04-13 19:05:35,940] torch.distributed.elastic.multiprocessing.api: [ERROR] failed (exitcode: 1) local_rank: 0 (pid: 1473334) of binary: /home/sharipov/monet/venv/bin/python
Traceback (most recent call last):
  File "/home/sharipov/monet/venv/bin/torchrun", line 8, in <module>
    sys.exit(main())
  File "/home/sharipov/monet/venv/lib64/python3.9/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 347, in wrapper
    return f(*args, **kwargs)
  File "/home/sharipov/monet/venv/lib64/python3.9/site-packages/torch/distributed/run.py", line 812, in main
    run(args)
  File "/home/sharipov/monet/venv/lib64/python3.9/site-packages/torch/distributed/run.py", line 803, in run
    elastic_launch(
  File "/home/sharipov/monet/venv/lib64/python3.9/site-packages/torch/distributed/launcher/api.py", line 135, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/sharipov/monet/venv/lib64/python3.9/site-packages/torch/distributed/launcher/api.py", line 268, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
train.py FAILED
------------------------------------------------------------
Failures:
  <NO_OTHER_FAILURES>
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2024-04-13_19:05:35
  host      : i01.izar.cluster
  rank      : 0 (local_rank: 0)
  exitcode  : 1 (pid: 1473334)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
